code/modules/cm_marines/dropship_ammo.dm

```diff
@@ -165,25 +167,27 @@
 	var/soundplaycooldown = 0
 	var/debriscooldown = 0
 	for(var/i = 1 to ammo_used_per_firing)
-		var/turf/U = pick(turf_list)
+		var/turf/Impact = pick(turf_list)
```

Comment:
We use snake_case so no capitals and more than one word should have a "_" instead of a space